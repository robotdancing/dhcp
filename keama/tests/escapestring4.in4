# string option-data with embedded commas

# authoritative is mandatory
authoritative;

# vendor option space
option a-string code 250 = text;

option a-string "foo, bar";
