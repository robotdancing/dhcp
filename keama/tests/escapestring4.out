{
  # string option-data with embedded commas
  # authoritative is mandatory
  "Dhcp4": {
    "option-def": [
      # vendor option space
      {
        "space": "dhcp4",
        "name": "a-string",
        "code": 250,
        "type": "string"
      }
    ],
    "option-data": [
      {
        "space": "dhcp4",
        "name": "a-string",
        "code": 250,
        "data": "foo\\, bar"
      }
    ]
  }
}
